dashboard/app/views/plc/courses/show.html.haml

```diff
@@ -1,6 +1,67 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+  = javascript_include_tag 'js/plc'
+
+
+:javascript
+  var allModuleNames = #{options_for_plc_task_learning_modules}
```

Comment:
Ah - this was part of a scheme to get this into the evaluation_creation.js but it isn't needed anymore - will remove it